admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1166,11 +1162,11 @@ class AdminModuleController {
       }
     }
 
-    function updateText(element, value) {
+    const updateText = (element, value) => {
       const explodedText = element.text().split(' ');
       explodedText[0] = value;
       element.text(explodedText.join(' '));
-    }
+    };
```

Comment:
As I said before, you'll have hoisting problems if you declare a function _after_ it's used. Please declare functions _before_ using them.

Reply:
Done, my bad!